packages/core/src/DdSdkReactNative.tsx

```diff
@@ -4,87 +4,111 @@
-                resolve()
-                return
+                InternalLog.log(
+                    "Can't initialize Datadog, SDK was already initialized",
+                    SdkVerbosity.WARN
+                );
+                resolve();
+                return;
             }
 
-            InternalLog.verbosity = configuration.verbosity
+            InternalLog.verbosity = configuration.verbosity;
 
-            configuration.additionalConfig[DdSdkReactNative.DD_SOURCE_KEY] = 'react-native';
-            configuration.additionalConfig[DdSdkReactNative.DD_SDK_VERSION] = sdkVersion;
-            configuration.additionalConfig[DdSdkReactNative.DD_NATIVE_VIEW_TRACKING_KEY] = configuration.nativeViewTracking;
+            configuration.additionalConfig[DdSdkReactNative.DD_SOURCE_KEY] =
+                'react-native';
+            configuration.additionalConfig[
+                DdSdkReactNative.DD_SDK_VERSION
+            ] = sdkVersion;
+            configuration.additionalConfig[
+                DdSdkReactNative.DD_NATIVE_VIEW_TRACKING_KEY
+            ] = configuration.nativeViewTracking;
             if (configuration.verbosity != undefined) {
-                configuration.additionalConfig[DdSdkReactNative.DD_SDK_VERBOSITY_KEY] = configuration.verbosity;
+                configuration.additionalConfig[
+                    DdSdkReactNative.DD_SDK_VERBOSITY_KEY
+                ] = configuration.verbosity;
             }
 
             if (configuration.proxyConfig) {
                 const additionalConfig = configuration.additionalConfig;
                 const proxyConfig = configuration.proxyConfig;
 
-                additionalConfig[DdSdkReactNative.DD_PROXY_TYPE_KEY] = proxyConfig.type;
-                additionalConfig[DdSdkReactNative.DD_PROXY_ADDRESS_KEY] = proxyConfig.address;
-                additionalConfig[DdSdkReactNative.DD_PROXY_PORT_KEY] = proxyConfig.port;
+                additionalConfig[DdSdkReactNative.DD_PROXY_TYPE_KEY] =
+                    proxyConfig.type;
+                additionalConfig[DdSdkReactNative.DD_PROXY_ADDRESS_KEY] =
+                    proxyConfig.address;
+                additionalConfig[DdSdkReactNative.DD_PROXY_PORT_KEY] =
+                    proxyConfig.port;
                 if (proxyConfig.username && proxyConfig.password) {
                     if (proxyConfig.type == ProxyType.SOCKS) {
-                        console.warn("SOCKS proxy configuration doesn't support Basic authentication.")
+                        console.warn(
+                            "SOCKS proxy configuration doesn't support Basic authentication."
+                        );
                     } else {
-                        additionalConfig[DdSdkReactNative.DD_PROXY_USERNAME_KEY] = proxyConfig.username;
-                        additionalConfig[DdSdkReactNative.DD_PROXY_PASSWORD_KEY] = proxyConfig.password;
+                        additionalConfig[
```

Comment:
yes this can happen. Actually I believe we see a lot of this because of our long names and OOP, and also we have a lot of `if` chained here. This is a good sign that there is something that can be improved.